### DISCUSSION TITLE
It is possible to have a discontinuous townblock without being an outpost?

### DISCUSSION THREAD

**Discussion by MinamiAkira** [TownyAdvanced/Towny#6326] [Q&A] [answered]
Example: I claim from point A to point B and after that, I unclaim at point C between point A and point B.

Map:

Before: H+A+C+B
After: H+A+-+B

"H" -> Homeblock
"+"  -> Townblock
"-"  -> Wildness
A, C, B -> Point A, Point C, Point B

If yes, is this a bug or is it on purpose?

**Comment by LlmDl** [comment, marked as answer]
It's not a bug. We don't want to track contiguous claims.
